index.md

```diff
@@ -12,7 +12,7 @@ layout: index
 
 		<a href="adult-hiv/0-3-contents.html" title="Adult HIV"><img class="cover-thumb" src="adult-hiv/images/cover-thumb.jpg" alt="Adult HIV" title="Adult HIV is for primary care nurses, medical officers and lay counsellors. It covers management of HIV-infected adults at primary-care clinics, preparing patients for antiretroviral (ARV) treatment, ARV drugs, starting and maintaining patients on ARV treatment and an approach to opportunistic infections." /></a>
 		
-		<a href="birth-defects/0-3-contents.html" title="Birth Defects"><img class="cover-thumb" src="birth-defects/images/cover-thumb.jpg" alt="Birth Defects" title="Birth Defects is for those who work with patients with birth defects. It covers modes of inheritance, medical genetic counselling, birth defects due to chromosomal abnormalities, single gene defects, teratogens and multifactorial inheritance."/></a>
+		<a href="congenital disorders/0-3-contents.html" title="Congenital Disorders"><img class="cover-thumb" src="congenital-disorders/images/cover-thumb.jpg" alt="Congenital Disorders" title="Congenital Disorders is for those who work with patients with congenital disorders. It covers modes of inheritance, medical genetic counselling, congenital disorders due to chromosomal abnormalities, single gene defects, teratogens and multifactorial inheritance."/></a>
```

Comment:
@nkoeries Typo here: you need a hyphen in `href="congenital-disorders"` for the link to work (i.e. click on cover > opens book). You can test the changes on your own fork by going to your fork's staging site at https://nkoeries.github.io/bettercare/

Can you fix that, test, and let me know if it's working?